robot/basestation/static/js/elements/navigation/localization.js

```diff
@@ -119,11 +126,10 @@ function check_rover_position() {
 
 // function which converts the values into something usable by the canvas
 function adjust_values_to_cartesian(rover_lat, rover_long, rover_heading) {
-    let angle = Direction(base_lat, base_long, rover_lat, rover_long)
-    let distance = Distance(base_lat, base_long, rover_lat, rover_long)
+    let direction = direction_to_rover(base_lat, base_long, rover_lat, rover_long)
+    let distance = distance_to_rover(base_lat, base_long, rover_lat, rover_long)
```

Comment:
this is better, i might use something like `antennaToRoverDir` for example to be even more clear with the variable names.